bundle/Templating/Twig/Extension/NamedObjectRuntime.php

```diff
@@ -24,23 +25,23 @@ class NamedObjectRuntime
     private $namedObjectProvider;
 
     /**
-     * @var \Psr\Log\LoggerInterface
+     * @var bool
      */
-    private $logger;
+    private $isDebug;
 
     /**
-     * @var bool
+     * @var \Psr\Log\LoggerInterface|null
      */
-    private $isDebug;
+    private $logger;
 
     public function __construct(
         Provider $specialObjectProvider,
-        LoggerInterface $logger,
-        bool $isDebug
+        bool $isDebug,
+        ?LoggerInterface $logger
```

Comment:
Since you now reordered the arguments, you may as well use `?LoggerInterface $logger = null` :+1:

Reply:
Updated: https://github.com/netgen/ezplatform-site-api/pull/164/commits/aafb17a6f4bf981956f28d351472966f352ecf2a